src/components/Post.js

```diff
@@ -151,18 +151,82 @@ export default class Post extends React.Component {
 const ExpandedPostDetails = props => {
   let {
     post, image, comments, commentsExpanded,
-    commentingDisabled, onCommentCreate, communities
+    commentingDisabled, onCommentCreate, communities, currentUser
   } = props
   let description = present(sanitize(post.description))
   let attachments = filter(post.media, m => m.type !== 'image')
   if (!comments) comments = []
 
+  var yeses = filter(post.responders, er => er.response === 'yes')
```

Comment:
Making components smaller seems to be a good rule of thumb in React.
On Fri, Dec 4, 2015 at 11:01 AM Robbie Carlton <notifications@github.com>
wrote:

> In src/components/Post.js
> <https://github.com/Hylozoic/hylo-redux/pull/2#discussion_r46718208>:
>
> >    } = props
> >    let description = present(sanitize(post.description))
> >    let attachments = filter(post.media, m => m.type !== 'image')
> >    if (!comments) comments = []
> >
> > +  var yeses = filter(post.responders, er => er.response === 'yes')
>
> @levity <https://github.com/levity> - there's so much code specifically
> for the event responses, I'm tempted to pull it out into it's own
> component. What are your thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Hylozoic/hylo-redux/pull/2/files#r46718208>.
>